database/seeds/ModelSeeders/UserSeeder.php

```diff
@@ -0,0 +1,120 @@
+<?php
+
+use Illuminate\Database\Seeder;
+use Faker\Factory as Faker;
+
+class UserSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        // DB::table('phpbb_users')->delete();
+        // DB::table('osu_user_stats')->delete();
+        // DB::table('osu_user_stats_fruits')->delete();
+        // DB::table('osu_user_stats_mania')->delete();
+        // DB::table('osu_user_stats_taiko')->delete();
+        // DB::table('osu_user_performance_rank')->delete();
+
+        $this->beatmapCount = App\Models\Beatmap::count();
+        $this->faker = Faker::create();
+        if ($this->beatmapCount === 0) {
+            $this->command->info('Can\'t seed events, scores, events or favourite maps due to having no beatmap data.');
+        }
+
+        $this->stats_count = 0;
+        $this->hist_count = 0;
```

Comment:
was meant to be used as a kind of check to make sure it was created and avoid errors, but I don't think it's necessary so i"ll remove it.